### DISCUSSION TITLE
How to fix user account block when logging in with github in vercel?

### DISCUSSION THREAD

**Discussion by SanahAZ** [vercel/community#2904] [Help] [answered]
### Summary

I've created a new github account and when i try to signup with vercel it displays that my account is blocked

![Screen Shot 2023-06-18 at 5 14 01 PM](https://github.com/vercel/community/assets/136971420/10b11cbf-1958-4ab3-a18f-0518d08c229e)

### Steps to Reproduce

I've created and deployed multiple projects on vercel with Github repository. and currently i've been working on a project and created a new account on github and pushed my code on my repository. after that i've tried to signup to my vercel with the new Github account. but when i do that it displays an error This user account is blocked. Contact registration@vercel.com for more information..

I have no idea as to why this is happening and how to fix this. can anyone please explain this and help me to fix this?

**Comment by amyegan** [comment, marked as answer]
If you have not already done so, please contact [registration@vercel.com](mailto:registration@vercel.com) as instructed in the error message 🙏

**Reply by noblez-90z** [reply]
I'm having same issues and i have contacted the above mentioned email but no positive feedback
